docs/reST/ref/event.rst

```diff
@@ -431,6 +431,8 @@ Most these window events do not have any attributes, except ``WINDOWMOVED``,
    Creates a new event with the given type and attributes. The attributes can
    come from a dictionary argument with string keys or from keyword arguments.
 
+   .. versionchanged:: 2.1.3 Event works the same way ``EventType`` does.
+
    .. ## pygame.event.Event ##
 
 .. class:: EventType
```

Comment:
It would be nicer if the docs for `Event` and `EventType` were unified here, and a note added that `EventType` is an alias for `Event` since 2.1.3